### ISSUE TITLE
Nuxeo API bug with “ancestor” query -- not retrieving content based on current folder location?

### ISSUE THREAD

**Issue by christinklez** [ucldc/nuxeo-extent-stats#4]

**Comment by barbarahui** [comment]
@christinklez @aturner I can't reproduce this bug. Do you it's worth asking UCI to walk us through how they moved the documents between folders so we can hone in on how this happened (like we did with UCB for the complex object component position bug)?

**Comment by christinklez** [comment]
We discussed that since staff at UCI have departed since the project folders were moved, we are unlikely to reproduce the move steps they had taken.

This counts bug occurs at the top-most hierarchy project folder (e.g., SCA_OCRH & SCA_SC) and does not seem to impact the sub-folders within, which is the level in which harvesting occurs. We can confirm that when we were rebuilding the initial Rikolti MVP index, we confirmed all record counts matched, or had checked in with UCI staff if any record counts did not line up.

Post-Nuxeo upgrade to LTS 2023, let's revisit to see if this bug still occurs during extent stats season 2025. Thanks!
